pkg/networkpolicy/controller.go

```diff
@@ -298,6 +326,75 @@ func (c *Controller) Run(ctx context.Context) error {
 	return nil
 }
 
+// evaluatePacket evalute the network policies using the following order:
+// 1. AdminNetworkPolicies in Egress for the source Pod/IP
+// 2. NetworkPolicies in Egress (if needed) for the source Pod/IP
+// 3. AdminNetworkPolicies in Ingress for the destination Pod/IP
+// 4. NetworkPolicies in Ingress (if needed) for the destination Pod/IP
+func (c *Controller) evaluatePacket(p packet) bool {
```

Comment:
@npinaeva please if this logic is correct now

Reply:
ref https://github.com/kubernetes-sigs/kube-network-policies/pull/9#discussion_r1577518558

Reply:
yeah I think looks correct now!